src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -149,90 +149,61 @@ them, and how to use them to configure your environment.
-
-    -   *INSTANCE_TYPE:* One of m1.small, m1.large, m1.xlarge,
-        c1.medium, c1.xlarge, which defines what kind of hardware the
-        instance will be running on. m1 instances are memory focused,
-        whereas c1 instances are for more CPU intensive tasks. This
-        property defaults is m1.small, the cheapest and slowest of all.
+    *Create Node Source* are:
+    
+    -   *rmDomain:* The public IP address of the resource manager. This IP needs to be accessible from the AWS cloud. 
+    
+    -   *connectorIaasURL:* Connector-iaas is a service embedded in the scheduler used to interact with IaaS like AWS. By default it runs on the same domain and port of the scheduler.    
+
+    -   *image:* Defines which AMI will be deployed on the AWS instances to create. The
+        value to provide is the AWS region together with the unique AMI Id, for example : eu-west-1/ami-bff32ccc. 
+        
+    -   *numberOfInstances:* Total number of AWS instances to create for this infrastructure. 
+    
+    -   *numberOfNodesPerInstance:* Total number of Proactive Nodes to deploy in each AWS created instance. 
+    
 +
-WARNING: Only m1.xlarge and c1.xlarge instances are able to launch
-x86_64 AMI. When trying to run such AMI, if INSTANCE_TYPE is
-not one of the xlarge, it will be forced to m1.xlarge. If you
-want to control the cost of your instances, do not run x86_64
-AMIs.
+TIP: If all the nodes of an AWS instance are removed, the instance will be terminated.
 +
-    -   *MAX_INST:* Sets a maximum number of instances that can be
-        deployed simultaneously on this infrastructure, which is useful
-        to control the cost of instances.
-
-Using the Scheduler and the Resource Manager with an EC2 NodeSource
-requires specific configuration. EC2 instances, indeed, are located on a
-private network, behind a NAT device: this allows fast communications
-between two EC2 instances, but this is an issue when using the RMI
-protocol. The workaround is to use a PAMR router, or the HTTP protocol.
-
-For that reason, the HTTP protocol is forced for communications going
-from the RM to the EC2 nodes. On the other way, from EC2 to the RM,
-using HTTP communications is also the simplest, also RMISSH is possible
-but requires manual configuration for deploying RSA keys. To configure
-your RM with HTTP, all you need to specify is a configuration file with
-the following properties, for both Resource Manager *and* Scheduler.
-
-``` {.xml}
-<prop key="proactive.net.nolocal" value="true"/>
-<prop key="proactive.communication.protocol" value="http" />
-<prop key="proactive.http.port" value="PORT" />
-<prop key="proactive.net.noprivate" value="true" />
-<prop key="proactive.useIPaddress" value="true" />
-```
-
-Note that you need to specify a different port for both configurations.
-Sample configurations can be found in */samples/ec2*.
+    
+    -   *operatingSystem:* The operating system running in the AWS instance.
```

Comment:
It is, but there is a different type of download command if it is windows or linux (wget or powershell....) and I cannot understand if it is windows from the image id. I guess this can and should be improved in the next iteration